lib/coursemology/evaluator/models/programming_evaluation.rb

```diff
@@ -30,7 +30,7 @@ def language=(language)
   def package
     @package ||= begin
       body = plain_request('courses/assessment/programming_evaluations/:id/package', id: id)
-      Package.new(StringIO.new(body))
+      Package.new(Coursemology::Evaluator::StringIO.new(body))
```

Comment:
Yah, I noticed that custom StringIO, but not sure RubyZip completely works with that. that' why I'm asking.